thunderfish/consistentfishes.py

```diff
@@ -1,18 +1,25 @@
+"""
+Functions to compare fishlists created by the harmonicgroups.py module in order to create a create a fishlist with only
+these fishes available in all fishlists.
+
+consistent_fishes(): Compares a list of fishlists and builds a consistent fishlist.
+"""
+
 import numpy as np
 import harmonicgroups as hg
 
 
 def find_consistency(fundamentals, df_th=1.):
     """
-    Compares several lists to find values that are, with a certain threshhold, available in every list.
+    Compares lists of floats to find these values consistent in every list. (with a certain threshold)
 
-    This function gets a list of lists conatining values as input. For every value in the first list this function is
-    looking in the other lists if there is a entry with the same value +- threshold. The vector consistancy_help
-    has the length of the first list of fundamentals and it's entries are all ones. When a value from the first list is
-    also available in another list this function adds 1 to that particular entry in the consostany_help vector. After
-    comparina all values from the first list with all values of the other lists the consostancy_help vector got values
-    between 1 and len(fundamentals). The indices where this vector is equal to len(fundamentals) are also the indices of
-    fishes in the first fishlist that are available in every fishlist.
+    Every value of the first list is compared to the values of the other lists. The consistency_help array consists in
```

Comment:
Delete the rest of the docu. it's actually easier to understand the code lines than the docu. (not your fault, it's simply hard to explain and it's not such long code.)